Update obstacles.js

MarcRFC opened the pull request:
A better approach for obstacle spawning. Still room for improvement, but at least no overlapping obstacles.

MarcRFC committed:
Update obstacles.js

baranidlo reviewed (changes requested):
Hey! Thanks for the commit.
But I can see that you compute px and px, but don't actually use it in the calculation of delta value at all.
Can you fix this, re-test and update the commit?
Thanks.

MarcRFC committed:
Update obstacles.js

I don't know what the purpose of PX[] and PY[] is. I got decent ostacle placement using this code.

MarcRFC commented:
I made the change

baranidlo commented:
Ok, that was not exactly what I had in mind, but let's give it a try..
